CHANGES.rst

```diff
@@ -1,21 +1,32 @@
 Changes
 =======
 
-Version 0.8.1 (UNRELEASED)
+Version 0.8.1 (2022-02-09)
 --------------------------
 
 - Users:
     - Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs.
+    - Adds Kubernetes job memory limits validation before publishing workflow submission.
+    - Adds support for HTML preview of workspace files in the user interface.
+    - Adds search by name in workflow file list page of the user interface.
+    - Changes user interface to display a more accurate Cluster Health status page, based on the availability instead of the usage.
```

Comment:
Changes the Cluster Health web interface page to display the cluster status information based on resource availability rather than only usage.